cl/simple_pages/templates/help/recap_email_help.html

```diff
@@ -0,0 +1,95 @@
+{% extends "base.html" %}
+
+{% block title %}Help with @recap.email – CourtListener.com{% endblock %}
+{% block footer-scripts %}
+  {% include "includes/anchors.html" %}
+{% endblock %}
+{% block sidebar %}{% endblock %}
+
+
+{% block content %}
+<div class="hidden-xs col-sm-2 col-md-3"></div>
+<div class="col-xs-12 col-sm-8 col-md-6">
+    <h1>Help with @recap.email</h1>
+    <p class="lead">@recap.email is a system we use to gather content from PACER and add it to the <a href="https://www.courtlistener.com/recap/">RECAP Archive</a>. If you receive notification emails from PACER, it only takes a minute to set up this system and begin contributing content to the public commons.
+    </p>
+
+    <h2>How it works</h2>
+    <h3>Three easy steps:</h3>
+    <ol>
+        <li><p>Every party in a case gets notification emails that contain one-time "magic links" that give free access to new filings in the case.</p></li>
+        <li><p>We give you an email address that you can configure in your PACER account as a secondary recipient for these notifications. Your primary address in PACER is never changed.</p>
+            {% if request.user.is_authenticated %}
+            <p><b>Your @recap.email address is {{ request.user.profile.recap_email }}</b></p>
+            {% else %}
+            <p>Log in to see your personal @recap.email address.</p>
+            {% endif %}
+        </li>
+        <li>After you add us as a secondary recipient in PACER, we'll get a copy of your notification emails. When we do, we'll add the items to RECAP and we'll send you a notification so you know we did.</li>
+    </ol>
+    <p>That's it!</p>
+
+    <h3>Why is this useful to me?</h3>
```

Comment:
```suggestion
    <h2>Why is this useful to me?</h2>
```

Reply:
The rest of these should go to h2 as well, or they should all be h3's if h2 is too large. But There's only two levels of hierarchy on the page: The title and then the questions.